tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -2378,22 +2378,18 @@ describe('scalar card', () => {
       }));
 
       it('does not render fobs when no timeSelection is provided', fakeAsync(() => {
-        store.overrideSelector(getMetricsLinkedTimeSelection, {
-          start: {step: 20},
-          end: null,
-        });
+        store.overrideSelector(getMetricsLinkedTimeSelection, null);
         store.overrideSelector(
           selectors.getIsLinkedTimeProspectiveFobEnabled,
           true
         );
         const fixture = createComponent('card1');
         fixture.detectChanges();
         const fobController = fixture.debugElement.query(
-          By.directive(CardFobComponent)
+          By.directive(CardFobControllerComponent)
         ).componentInstance;
 
-        expect(fobController.startFobWrapper).toBeUndefined();
-        expect(fobController.endFobWrapper).toBeUndefined();
+        expect(fobController).toBeDefined();
```

Comment:
Is the assert corresponding to the test? ("does not render fobs when no timeSelection is provided")
Can we add 2395-2396 back?

Reply:
I felt like those were kinda redundant with other tests but I can definitely add them back.